src/components/Contributing.jsx

```diff
@@ -0,0 +1,61 @@
+import { ContributorList } from './ContributorList';
+import { DocsList } from './DocsList';
+
+export function Contributing() {
+  return (
+    <section id="contributing" style={{ padding: '4rem 0' }}>
+      <div className="mzp-l-content mzp-t-content-xl">
+        <article className="mzp-c-article">
+          <h2 className="mzp-c-article-title" style={{ marginBottom: '2rem' }}>
+            Contributing
+          </h2>
+          <p>
+            Contributions are welcome, and they are greatly appreciated! Every
+            little bit helps, and credit will always be given.
```

Comment:
Leaving a comment to convert to an issue for me to solve later:

Add a brief "5 minute contribution" (bug reports), "30 minute contribution" (add documentation), "weekend contribution" (add a minor feature, fix a bug), "longer" (larger projects, etc.)